analytics.js

```diff
@@ -1337,10 +1337,20 @@ const eventsPushedAlready =
   window.rudderanalytics.push == Array.prototype.push;
 
 const argumentsArray = window.rudderanalytics;
-
-while (argumentsArray && argumentsArray[0] && argumentsArray[0][0] !== "load") {
-  argumentsArray.shift();
+/**
+ * Iterate the call stack until we find load call and
+ * then shift it to the beginning.
+ *
+ * Ex: Say the call stack currently have [page, identify, load, track]
+ * It will become [load, page, identify, track]
+ */
+for (let i = 0; i < argumentsArray.length; i++) {
+  if (argumentsArray[i] && argumentsArray[i][0] === "load") {
+    argumentsArray.unshift(argumentsArray.splice(i, 1)[0]);
+    break;
+  }
```

Comment:
If no `load` call was found, others calls should not be processed.

Reply:
There might be scenarios where the load method is called on button click/ called from the body, not from the head tag. For such cases load won’t be present in argumentsArray. We should let other calls to be processed.

Reply:
Addressed